tests/test_vcs_hg/code/Colon: name

```diff
@@ -1 +0,0 @@
-Filename has a colon
```

Comment:
Must be a Windows thing because that file's named like that... I'll try repushing this from linux.